.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -66,6 +75,52 @@ wait_for_reboot() {
 	done
 }
 
+wait_mcp_update() {
+	local delta="${1:-900}"
+	local sleep_time=60
```

Comment:
That's a pretty long sleep!

Reply:
@jodh-intel ,

Have you ever watched TV series like [Pawn Stars](https://www.imdb.com/title/tt1492088/)? At some point they "meet in the middle" and shake hands. So 30s of sleep time works for you? :)